### PR TITLE
Support [--yes] parameter when upgrade cluster

### LINKED ISSUE #793: Support [--yes] parameter when upgrade cluster
### Your current KubeKey version

version.BuildInfo{Version:"v1.1.1-alpha.1", GitCommit:"9cc863e24df23f3f7f41fed8be2c85604c5c569b", GitTreeState:"dirty", GoVersion:"go1.14.14"}

### Describe this feature

when upgrade cluster, we need confirm yes or no.

```
root@master1:~# ./kk upgrade -f config-sample.yaml
+-------------+------+------+---------+----------+-------+-------+-----------+---------+------------+-------------+------------------+--------------+
| name        | sudo | curl | openssl | ebtables | socat | ipset | conntrack | docker  | nfs client | ceph client | glusterfs client | time         |
+-------------+------+------+---------+----------+-------+-------+-----------+---------+------------+-------------+------------------+--------------+
| worker-s001 | y    | y    | y       | y        | y     | y     | y         | 20.10.7 | y          | y           | y                | CST 11:29:10 |
| worker-s002 | y    | y    | y       | y        | y     | y     | y         | 20.10.7 | y          | y           | y                | CST 11:29:10 |
| master1     | y    | y    | y       | y        | y     | y     | y         | 20.10.7 | y          | y           | y                | CST 11:29:10 |
+-------------+------+------+---------+----------+-------+-------+-----------+---------+------------+-------------+------------------+--------------+

Cluster nodes status:
NAME          STATUS   ROLES                         AGE   VERSION
master1       Ready    control-plane,master,worker   16m   v1.20.6
worker-s001   Ready    worker                        16m   v1.20.6
worker-s002   Ready    worker                        16m   v1.20.6

Upgrade Confirmation:
kubernetes version: v1.20.6 to v1.20.6

Continue upgrading cluster? [yes/no]: ^C

```

### Describe the solution you'd like

maybe add a [--yes] parameter to skip this input.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by chaunceyjiang**
Support [--yes] parameter when upgrade cluster

Fix #793

### CODE REVIEW

**Comment by 24sama** [comment]
Thanks for your contribution.

File: pkg/pipelines/upgrade_cluster.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func NewUpgradeClusterPipeline(runtime *common.KubeRuntime) error {
 	m := []module.Module{
 		&precheck.NodePreCheckModule{},
 		&precheck.ClusterPreCheckModule{},
-		&confirm.UpgradeConfirmModule{},
+		&confirm.UpgradeConfirmModule{Skip: runtime.Arg.SkipConfirmCheck},
```
Comment (24sama) [inline pkg/pipelines/upgrade_cluster.go:37]: This variable hasn't been set, you need to set this variable on https://github.com/kubesphere/kubekey/blob/b5a41a021d49cb1bf90d2e2c8a3f042b87e7863a/cmd/ctl/upgrade/upgrade.go#L84

**Review by 24sama — CHANGES_REQUESTED** [review]

File: pkg/pipelines/upgrade_cluster.go (hunk 1 above)
Comment (chaunceyjiang) [reply pkg/pipelines/upgrade_cluster.go:37]: ok

**Review by 24sama — COMMENTED** [review]
done.

**Pull request merged**